coalib/results/result_actions/ShowPatchAction.py

```diff
@@ -113,8 +113,10 @@ def apply(self,
 
         for filename, this_diff in sorted(result.diffs.items()):
             to_filename = this_diff.rename if this_diff.rename else filename
+            to_filename = this_diff.create if this_diff.create else to_filename
             to_filename = '/dev/null' if this_diff.delete else to_filename
-            original_file = original_file_dict[filename]
+            original_file = ([] if this_diff.create
```

Comment:
It'd raise a `KeyError`. Since the file being created didn't exist in the FS previously, and `original_file_dict` contains the files that were read/loaded by coala. So I don't think this works..